### ISSUE TITLE
Missing license or copyright

### ISSUE THREAD

**Issue by kaby76** [antlr/grammars-v4#2535]
As per a conversation in [the antlr-discussions Google group](https://groups.google.com/g/antlr-discussion/c/-g5CF0MuUBk), there is confusion as to use of the grammars in this Github repo. I did a check to see what contains either the word "copyright" or "license" on the grammars here, and found that there are quite a few missing attribution. There may also be problems in attribution of the grammar due to scraping it from another copyrighted source.

    grep -L -i -e copyright -e license `find . -name '*.g4' | grep -v Generated | grep -v examples`

	./abb/abbLexer.g4
	./algol60/algol60.g4
	./apt/apt.g4
	./asm/masm/MASM.g4
	./bcpl/bcpl.g4
	./bcpl/origbcpl.g4
	./capnproto/CapnProto.g4
	./clojure/Clojure.g4
	./cpp/CPP14Lexer.g4
	./cql3/CqlLexer.g4
	./css3/css3.g4
	./dcm/DCM_2_0_grammar.g4
	./flatbuffers/FlatBuffers.g4
	./fol/fol.g4
	./gdscript/GDScriptLexer.g4
	./gdscript/GDScriptParser.g4
	./http/http.g4
	./informix/informix.g4
	./javascript/typescript/TypeScriptLexer.g4
	./jpa/JPA.g4
	./json/JSON.g4
	./json5/JSON5.g4
	./kirikiri-tjs/TJSLexer.g4
	./kirikiri-tjs/TJSParser.g4
	./kotlin/kotlin/KotlinLexer.g4
	./kotlin/kotlin/KotlinParser.g4
	./kotlin/kotlin/UnicodeClasses.g4
	./kotlin/kotlin-formal/KotlinLexer.g4
	./kotlin/kotlin-formal/KotlinParser.g4
	./kotlin/kotlin-formal/UnicodeClasses.g4
	./lark/LarkLexer.g4
	./lark/LarkParser.g4
	./lpc/LPC.g4
	./mckeeman-form/McKeemanForm.g4
	./mdx/mdx.g4
	./mps/mps.g4
	./mumath/mumath.g4
	./oncrpc/oncrpcv2.g4
	./oncrpc/xdr.g4
	./pddl/Pddl.g4
	./powerbuilderdw/PowerBuilderDWParser.g4
	./powerquery/PowerQueryLexer.g4
	./powerquery/PowerQueryParser.g4
	./properties/properties.g4
	./python/python2/Python2.g4
	./python/python2-js/Python2.g4
	./racket-bsl/BSL.g4
	./racket-isl/ISL.g4
	./rexx/RexxLexer.g4
	./rexx/RexxParser.g4
	./sharc/SHARCLexer.g4
	./sharc/SHARCParser.g4
	./smalltalk/Smalltalk.g4
	./sql/postgresql/PostgreSQLParser.g4
	./stacktrace/StackTrace.g4
	./stellaris/stellaris.g4
	./stl/STL.g4
	./swift/swift5/Swift5Lexer.g4
	./swift/swift5/Swift5Parser.g4
	./teal/Teal.g4
	./thrift/Thrift.g4
	./tnsnames/tnsnamesParser.g4
	./unicode/graphemes/Graphemes.g4
	./unreal_angelscript/UnrealAngelscriptLexer.g4
	./vb6/VisualBasic6Lexer.g4
	./velocity/VTLLexer.g4
	./velocity/VTLParser.g4
	./wavefront/WavefrontOBJ.g4
	./wkt/wkt.g4
	./xpath/xpath20/XPath20.g4
	./xpath/xpath31/XPath31.g4
	./_scripts/templates/Arithmetic.g4

**Comment by RossPatterson** [comment]
./rexx/*.g4 were fixed by [commit 4cbcf36236d7e30e3570e2f2f7def50f8253866d]
(https://github.com/antlr/grammars-v4/commit/4cbcf36236d7e30e3570e2f2f7def50f8253866d) on 2022-02-28..

**Comment by kaby76** [comment]
> ./rexx/*.g4 were fixed by [commit 4cbcf36236d7e30e3570e2f2f7def50f8253866d](https://github.com/antlr/grammars-v4/commit/4cbcf36236d7e30e3570e2f2f7def50f8253866d) on 2022-02-28..

That's true, but there's ` find . -name '*.g4' | grep -v examples | wc` => 426 grammar files that all need to be checked.
